bentoml/server/bento_api_server.py

```diff
@@ -172,8 +200,29 @@ def setup_routes(self):
         /classify
         /predict
         """
+        if self.static_path:
+            self.app.add_url_rule(
```

Comment:
Could you add some inline comments about these three `url_rule` here? reading this code it is not immediately clear to me what it does

Reply:
Ah, sorry...this is a neccessary evil😅 as flask doesn't handle it automagically.
```python
            # serve static files for any given path
            # this will also serve index.html from directory /any_path/
            # for path as /any_path/
            self.app.add_url_rule(
                "/<path:file_path>",
                "static_proxy",
                partial(self.static_serve, self.static_path),
            )
            # serve index.html from the directory /any_path
            # for path as /any_path/index
            self.app.add_url_rule(
                "/<path:file_path>/index",
                "static_proxy2",
                partial(self.static_serve, self.static_path),
            )
            # serve index.html from root directory for path as /
            self.app.add_url_rule(
                "/", "index", partial(self.index_view_func, self.static_path)
            )
```
Any changes to this? Sorry but I am still trying to improve my commenting and documentation skills.

Reply:
@Korusuke that looks great, adding a little context here helps a lot!